cubic-server/World.cpp

```diff
@@ -103,6 +103,37 @@ void World::forEachEntityIf(std::function<void(Entity *)> callback, std::functio
         _dimension.second->forEachEntityIf(callback, predicate);
 }
 
+void World::forEachPlayer(std::function<void(Player *)> callback)
+{
+    std::vector<Player *> _players = this->getPlayers();
```

Comment:
_players is not a private variable from World class, so you just have to call it "players" without the underscore